source/jormungandr/jormungandr/interfaces/v1/journey_common.py

```diff
@@ -513,7 +513,15 @@ def __init__(self, output_type_serializer):
 
         parser_get.add_argument(
             "criteria",
-            type=OptionValue(['classic', 'robustness', 'occupancy']),
+            type=OptionValue(
+                [
+                    'classic',
+                    'robustness',
+                    'occupancy',
+                    'arrival_stop_attractivity',
+                    'departure_stop_attractivity',
+                ]
+            ),
```

Comment:
Shouldn't we hide the param for now and make it beta?

Reply:
I documented that the `criteria` feature is in beta